dev/com.ibm.ws.microprofile.config.1.3/test/src/com/ibm/ws/microprofile/config/loader/test/ServiceLoaderConfigSource.java

```diff
@@ -1,51 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *******************************************************************************/
-package com.ibm.ws.microprofile.config.loader.test;
-
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
-
-import org.eclipse.microprofile.config.spi.ConfigSource;
-
-@SuppressWarnings("serial")
-public class ServiceLoaderConfigSource extends ConcurrentHashMap<String, String> implements ConfigSource {
```

Comment:
I see these same set of classes have been deleted across 1.2, 1.3 and 1.4 unit tests. Is this just because they were unused?

Reply:
i deleted them from 1.4 but that broke unit tests running in eclipse. the fix was a bit ickie so i didn't bother trying for 1.2 or 1.3.